integration/utils/contLoadGenerator.go

```diff
@@ -40,15 +40,13 @@ type ContLoadGenerator struct {
 	initialized          bool
 	tps                  int
 	numberOfAccounts     int
-	trackTxs             bool
 	flowClient           *client.Client
 	serviceAccount       *flowAccount
 	flowTokenAddress     *flowsdk.Address
 	fungibleTokenAddress *flowsdk.Address
 	favContractAddress   *flowsdk.Address
 	accounts             []*flowAccount
-	availableAccounts    chan *flowAccount // queue with accounts available for   workers
-	txTracker            *TxTracker
+	availableAccounts    chan *flowAccount // queue with accounts available for workers
```

Comment:
can you remove the unused fields?  e.g., initialized & feedbackEnabled are populated, but never used.

Reply:
nice catch!  what static analyzer did you use to find these?

Reply:
cmd+f on chrome =P